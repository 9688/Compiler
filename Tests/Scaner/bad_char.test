'
'

'\0FF'
'\x0GG'
