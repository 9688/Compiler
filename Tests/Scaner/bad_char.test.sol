Ошибка в строке 1 позиции 2: отсутствует " ' "
Ошибка в строке 2 позиции 2: отсутствует " ' "
Ошибка в строке 4 позиции 3: недопустимый символ "5" для основания "8"
DOUBLE 4 4 5
Ошибка в строке 4 позиции 6: отсутствует " ' "
Ошибка в строке 5 позиции 3: отсутствует " ' "
IDENTIFICATOR 5 4 FF
Ошибка в строке 5 позиции 7: отсутствует " ' "
Ошибка в строке 6 позиции 4: отсутствует " ' "
IDENTIFICATOR 6 5 GG
Ошибка в строке 6 позиции 8: отсутствует " ' "
EOF 7 1 
