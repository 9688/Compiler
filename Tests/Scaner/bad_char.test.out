Ошибка в строке 1 позиции 2: отсутствует " ' "
Ошибка в строке 2 позиции 2: отсутствует " ' "
Ошибка в строке 4 позиции 3: недопустимый символ "F" для основания "8"
Ошибка в строке 5 позиции 4: отсутствует " ' "
EOF 6 1 
